docs/deprecated.md

```diff
@@ -19,6 +19,12 @@ The following list of features are deprecated in Engine.
 To learn more about Docker Engine's deprecation policy,
 see [Feature Deprecation Policy](https://docs.docker.com/engine/#feature-deprecation-policy).
```

Comment:
The policy needs an update. For changing the "3 stable releases", an easy fix is: "unless specified explicitly otherwise". Or we decide on making it always explicit. cc @thaJeztah

Reply:
https://github.com/docker/docker.github.io/pull/8960